packages/strapi-design-system/src/DesignSystemProvider.tsx

```diff
@@ -0,0 +1,43 @@
+import { createContext } from './helpers/context';
+import { ThemeProvider, ThemeProviderProps } from './ThemeProvider';
+
+const DEFAULT_LOCALE = 'en-EN';
+
+const getDefaultLocale = () => {
+  if (typeof navigator === 'undefined') {
+    return DEFAULT_LOCALE;
+  }
```

Comment:
This was already here, I assume because this would work in SSR envs, I don't see a problem keeping it for now, makes it easier if we _do_ move to SSR over the next months?